multigeiger/multigeiger.ino

```diff
@@ -242,7 +247,7 @@ void loop() {
   // main program: all other timestamp bookkeeping values shall be derived from it.
   // ISR code: only one timestamp shall be kept/updated there with the only purpose
   //           of being used to update the master timestamp.
-  static unsigned long gm_count_timestamp = 0;
+  static unsigned long gm_count_timestamp;
 
   // time between last 2 geiger mueller events [us]
   unsigned int gm_count_time_between;
```

Comment:
like **between**, the **timestamp** is initialized by the code before first usage.